canopy_dispatch.ml

```diff
@@ -66,67 +66,72 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     | uri::[] when uri = config.Canopy_config.push_hook_path ->
       store.update () >>= fun l ->
       respond_update l
+    | "tags"::[] -> (
+      let tags = Canopy_content.tags !cache in
+      let content = Canopy_article.to_tyxml_tags tags in
+      respond_html ~headers ~title:config.Canopy_config.blog_name ~content ~updated
```

Comment:
I am happy to change the code to use `store.last_commit () >>= fun updated` for this PR, but what if the last commit was not a new tag but a new article ? Shouldn't the Etag (for the `/tags` page) reflects the set of existing tags only ?